dataprofiler/data_readers/avro_data.py

```diff
@@ -73,19 +91,19 @@ def is_match(cls, file_path, options=None):
             options = dict()
 
         # get current position of stream
-        if data_utils.is_stream_buffer(file_path):
+        if data_utils.is_stream_buffer(file_path) and not isinstance(file_path, str):
```

Comment:
why this add?

Reply:
For the operation within the if statement, `file_path` can't be a string. `is_stream_buffer` checks if it's a file buffer but mypy doesn't seem to be able to detect it because it would complain on the next line that `.tell()` doesn't exist for strings. So far the only solution I could find was explicitly telling mypy it wouldn't be a string.

Reply:
outdated... no longer moot to this PR. already in `main`

Reply:
Can re remove in this PR?

Reply:
There should be a way to resolve this correctly. just need to find the right answer.

Reply:
agree